css/style.css

```diff
@@ -0,0 +1,658 @@
+  margin-top: 60px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+h1 {
+  font-size: 40px;
+  text-align: center;
+}
+
+/* Navbar Section */
+
+.navbar {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  height: 42px;
+  margin-left: 0;
+  border-bottom: 1px #e2e2e2 solid;
+  z-index: 1;
+  top: 0;
+}
+
+.left-logos {
+  display: flex;
+  align-items: center;
+  margin-left: 8%;
+}
+
+.left-logos button {
+  background-color: transparent;
+  border: 0;
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.main-logo {
+  margin-left: 96.594px;
+}
+
+.main-logo img {
+  padding-top: 10px;
+  width: 200px;
+  height: 25px;
+}
+
+.right-buttons {
+  display: flex;
+  margin-right: 8%;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.right-buttons button {
+  cursor: pointer;
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 3px;
+  text-align: center;
+  padding: 10px 15px 8px 15px;
+  border: 0;
+  margin-left: 10px;
+  font-size: 10.5px;
+}
+
+.right-buttons button:hover {
+  background-color: rgb(50, 104, 145);
+}
+
+/* End of Navbar Section */
+
+/* Main Page Section */
+
+.main-section {
+  width: 62.5%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main-tittle {
+  width: 600px;
+  padding: 0;
+}
+
+.main-tittle h1 {
+  text-align: left;
+}
+
+.article-author {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.article-author:hover {
+  text-decoration: none;
+}
+
+.text-box {
+  width: 50%;
+}
+
+.footer-tittle a {
+  color: rgb(86, 123, 149);
+  font-size: 11px;
+}
+
+.corporight a {
+  color: #666;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  padding: 8px 0;
+  font-size: 11px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.top-grid a,
+.bottom-grid a {
+  width: 94.5%;
+  text-decoration: none;
+}
+
+.text-box p a {
+  color: #326891;
+}
+
+.text-box p {
+  font-size: 1.25rem;
+  line-height: 1.875rem;
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  color: #333;
+}
+
+.image-container {
+  margin-bottom: 30px;
+}
+
+.image-container img {
+  width: 100%;
+  height: auto;
+}
+
+.image-container div {
+  width: 80%;
+}
+
+.main-section-footer-text p {
+  color: rgb(102, 102, 102);
+  font-size: 0.75rem;
+  line-height: 1rem;
+}
+
+.bottom-navbar-offer p {
+  font-size: 16px;
+  text-decoration: none;
+  color: whitesmoke;
+  cursor: pointer;
+}
+
+.subscribe div p {
+  text-align: center;
+  margin: 5px;
+  font-size: 16px;
+}
+
+.img-text div a {
+  color: rgb(51, 51, 51);
+  font-size: 14px;
+  line-height: 17px;
+  text-decoration: none;
+  font-weight: 500;
+  padding-bottom: 0;
+  border-bottom: none;
+  margin-left: 5px;
+  align-self: center;
+}
+
+.bottom-navbar-text p a {
+  font-size: 16px;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  You might directl going to `.bottom-navbar-text a { }` and  `.img-text a { }` delete other unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_